project-generator/src/test/java/io/syndesis/project/converter/DefaultProjectGeneratorTest.java

```diff
@@ -48,6 +48,7 @@
             System.setProperty("ivy.message.logger.level", "3");
     }
 
+    private static final String CONNECTOR_VERSION="0.4.5";
```

Comment:
completely agreed that it makes totally sense to extract this as constant. However it would be awesome if this property could be read from property which has been filter-copied by Maven with replacing the version with the current connector version used in the pom.xml. Otherwise its 100% guaranteed that this value will not be updated when connectors get updated.